content/200-concepts/100-components/02-prisma-client/051-working-with-advanced-types.mdx

```diff
@@ -11,6 +11,90 @@ This section describes how to work with field types like `Json` and lists/arrays
 
 </TopBlock>
 
+## Working with <inlinecode>Decimal</inlinecode>
```

Comment:
Maybe an intro sentence here?

Reply:
Argh, I forgot to finish this! Thank you!